test/functional/feature_bip68_sequence.py

```diff
@@ -33,6 +33,7 @@ def set_test_params(self):
             [
                 "-acceptnonstdtxn=1",
                 "-peertimeout=9999",  # bump because mocktime might cause a disconnect otherwise
+                "-pingtimeout=9999",  # bump because mocktime might cause a disconnect otherwise
```

Comment:
Previously, when ping timeouts were handled in `CConnman::InactivityCheck()`, the whole logic was gated on the following guard:

```
    if (now <= node.nTimeConnected + m_peer_connect_timeout) {
        // Only run inactivity checks if the peer has been connected longer
        // than m_peer_connect_timeout.
        return false;
    }
```

Because tests generally run for less than m_peer_connect_timeout, the ping timeout logic was not hit at all. Once the logic moves to net_processing, that guard no longer exists, so we start testing for ping timeouts as soon as the peer is connected.